src/core/lib/resolver/resolver_factory.h

```diff
@@ -68,7 +68,7 @@ class ResolverFactory {
   /// Returns a string representing the default authority to use for this
   /// scheme.
```

Comment:
Can we add a comment here that indicates that by default we URL encode the path?

Reply:
Done.